app/services/release_manager.rb

```diff
@@ -29,7 +28,16 @@ def create_release
 
   private
 
-  attr_reader :board, :repo, :release_branch_name, :extra_branches, :merge_conflicts
+  attr_reader :board, :repo, :release_branch_name, :merge_conflicts
+
+  def extra_branches
+    @extra_branches ||= 
+      if board.additional_branches_regex.present?
+        repo.regex_branches(Regexp.new(board.additional_branches_regex))
```

Comment:
- Guard against `RegexpError` being raised by invalid regexes? Add a validation for this when `Board` is saved?
- Strip `/` from start and end string in case they're present (since otherwise they're included as a literal "/")?